backends/ent/annotations.go

```diff
@@ -114,9 +117,17 @@ func (backend *Backend) ClearDocumentAnnotations(documentIDs ...string) error {
 		return nil
 	}
 
+	docUUIDs, err := backend.client.Document.Query().
+		QueryMetadata().
+		Where(metadata.NativeIDIn(documentIDs...)).
+		IDs(backend.ctx)
```

Comment:
This would return metadata UUIDs. I think it should instead be something like

```suggestion
	var docUUIDs []uuid.UUID

	docUUIDs, err := backend.client.Metadata.Query().
		Where(metadata.NativeIDIn(documentIDs...)).
		Select(node.FieldDocumentID).
		Scan(backend.ctx, &docUUIDs)
```

Reply:
Doesn't seem to like the `node.FieldDocumentID`, returns: 

```bash
--- FAIL: TestAnnotationsSuite/TestBackend_ClearDocumentAnnotations (1.41s)
    annotations_test.go:162: 
    Error Trace:    /Users/e386920/Documents/Github/storage/backends/ent/annotations_test.go:162
    Error:          Received unexpected error:
                    querying document IDs: ent: invalid field "document_id" for query
    Test:           TestAnnotationsSuite/TestBackend_ClearDocumentAnnotations
```

Got it working with: 

```go
	docUUIDs, err := backend.client.Metadata.Query().
		Where(metadata.NativeIDIn(documentIDs...)).
		QueryDocument().
		IDs(backend.ctx)
```

Reply:
Yeah copy paste error, whoops! Looks good